### PR TITLE
Make the first run wizard more helpful

### PR DESCRIPTION
**Pull request by juliushaertl**
This PR aims to add some more information to the firstrun wizard, to help users getting started.

Todo:
- [x] Make apps management page only show when the user is an admin
- [x] Use branded name from theming
- [x] Add indicator on what page you are

Input from @nextcloud/designers and maybe @jospoortvliet on the content would be nice. I mainly used some content from the website. Maybe you have some ideas for images/illustrations that would fit better.

We could also add some info pages about the groupware/talk features as well as some new features slide for what is added in major releases. Ideas for that are welcome.

cc @karlitschek 

Updated content:

## First page with values

image

## Details about clients

image

## Information on apps (for admins only)

image

## More information (help, community, enterprise support)

image

### CODE REVIEW

**Comment by enoch85** [comment]
Woop woop!

**Comment by karlitschek** [comment]
@schiessle

**Comment by karlitschek** [comment]
Very nice! We have to make sure that the messaging we send is correct. 
'.. on a server, at home, at one of our providers, or in a data center you know' I suggest to extend this to: '.. on a server, in your company, at home, at one of our providers, or in a data center you know'. 
On the first page we should list some of our values. So instead of the many links to sync clients, groupware, stay in touch and others (which we could move to a different page if we want, or not) we should list our values:
- Host you data and files where you decide
- 100% Open Source
- Open Standards and Interoperability
- Community focus

@jospoortvliet What do you suggest here? We need a message that works for community and companies.

@juliushaertl I would suggest to kill the 'Show First Run Wizard again' Button in the Settings and add an 'About' menu entry to the Dropdown menu between 'Help' and 'Log out'

**Comment by MariusBluem** [comment]
Could we make the content (texts, ...) themeable? Especially for enterprise users, sections like „Contributing“ do not make sense and could be replaced by a „Need help? Call IT departement for help“-section or whatever they want :)

**Comment by juliushaertl** [comment]
> On the first page we should list some of our values. 

Good idea.

> I would suggest to kill the 'Show First Run Wizard again' Button in the Settings and add an 'About' menu entry to the Dropdown menu between 'Help' and 'Log out'

I'm not sure if it should be that prominent, how if we just move it to the help section as kind of a start page for the manuals? But yes, we should remove it from the sync clients page, since that is not really related.

> Could we make the content (texts, ...) themeable? Especially for enterprise users, sections like „Contributing“ do not make sense and could be replaced by a „Need help? Call IT departement for help“-section or whatever they want :)

:+1: I'll have a look if we could use some existing custom theme mechanism there.

**Comment by karlitschek** [comment]
@juliushaertl We want/need some kind of About page anyways. We need to explain better what Nextcloud is, whats special about it and communicate our mission. The idea is to use the first run wizard for that. So it would be great if you could add the 'About' menu entry.

**Comment by jancborchardt** [comment]
I have to say that from a UX perspective this is all a bit TL;DR. :\ It’s 4 pages of really _a lot_ of text which keep me from simply using my Nextcloud. And most of the things which are written in there I actually already know once I made an account. I mean – the 1st page alone has _5_ links.

Sorry if this sounds negative. Onto constructive criticism:
We should better do it in a way like iOS apps and Android apps do it: Respect people’s time, have 3 pages max, very simple illustration/graphic and very little text on each page.

**Comment by schiessle** [comment]
@juliushaertl I really like your idea, also to have more than just one page to navigate through with a next button. I think having some core values on the first page and keeping it really clear makes absolutely sense. I also think it makes sense to have a about button which brings back the dialog. I think that how it works in many other solutions as well.

If this becomes more like a "About" page, which I would support I would also consider mention the license and link directly to the source code somewhere.

**Comment by schiessle** [comment]
For the core values, I think it would be nice to have one nice term for each, with a emblem and maybe two or three sentence below to describe what it actually means. Something like here https://userdatamanifesto.org

**Comment by juliushaertl** [comment]
> We want/need some kind of About page anyways. We need to explain better what Nextcloud is, whats special about it and communicate our mission. The idea is to use the first run wizard for that. So it would be great if you could add the 'About' menu entry.

Ok, if there will be some more general information that makes sense to me.

> I have to say that from a UX perspective this is all a bit TL;DR. :\ It’s 4 pages of really a lot of text which keep me from simply using my Nextcloud. And most of the things which are written in there I actually already know once I made an account. I mean – the 1st page alone has 5 links.

Sure we should keep that at a minimum but at least provide some information about nextcloud and on how you can get started. I'm fine to reduce the amount of pages. Also :+1: for going for a simpler text and just provide links for more information/to the documentation.

> For the core values, I think it would be nice to have one nice term for each, with a emblem and maybe two or three sentence below to describe what it actually means. 

Do we have them defined somehwere already, so I can have a look to integrate them nicely?

**Comment by jancborchardt** [comment]
An important thing to actually do here in the first run screen is to directly get a new person to put in their name, email address (if not there yet), and avatar. We should have these fields directly there so they don’t have to go to the Personal settings.

**Comment by juliushaertl** [comment]
> An important thing to actually do here in the first run screen is to directly get a new person to put in their name, email address (if not there yet), and avatar. We should have these fields directly there so they don’t have to go to the Personal settings.

Yes, that sounds like a good idea if we just show it if there is no data already set from an external user backend like LDAP.

**Comment by karlitschek** [comment]
really nice 👍

**Comment by jancborchardt** [comment]
Slide 1: 100% Open Source & community-focused can be combined. Making it a nice simple 3-column layout.

Slide 2: The headings and buttons should be center-aligned, looks much better balanced for a centered modal lile this.

Slide 3: No blue background around the image please, as it makes it way too small. First text block can be shortened to: "Find more than 100 apps in the Nextcloud app store to customize your cloud:"

Slide 4: Let's maybe make this a 3-column layout with simple icon-heading-shorttext? Colums would be "More info", " Get involved" and "Enterprise support".


Also, we should really have that ask for real name, email and avatar, if applicable.

**Comment by juliushaertl** [comment]
Thanks for the comments @jancborchardt I adjusted the PR accordingly. I will look into the displayname setup in a separate PR, so we can get this in for now.

Ready for review @nextcloud/designers

File: templates/page.apps.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+?>
+<div class="page" data-title="Extend your cloud" data-subtitle="">
+	<div class="image"><img src="<?php p(image_path('firstrunwizard', 'appstore.svg')); ?>" /></div>
+	<div class="description">
+		<p><?php p($l->t('Find more than 100 apps in the Nextcloud app store to customize your cloud:')); ?></p>
+		<ul>
+			<li><?php p($l->t('Groupware apps like Calendar, Contacts, Mail')); ?></li>
+			<li><?php p($l->t('Communication with Nextcloud Talk')); ?></li>
+			<li><?php p($l->t('Collaboration apps for document editing')); ?></li>
+			<li><?php p($l->t('Security and authentication features like two-factor authentication, SSO and randsomeware protection')); ?></li>
```
Comment (MariusBluem) [inline templates/page.apps.php]: Typo: `randsomeware` -> `ransomware`

**Comment by juliushaertl** [comment]
Fixed the typo and the lowercase letter in "a safe home for your data" as well.

**Pull request merged**